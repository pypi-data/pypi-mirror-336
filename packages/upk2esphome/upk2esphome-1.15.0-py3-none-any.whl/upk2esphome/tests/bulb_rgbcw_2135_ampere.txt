{
		"Jsonver": "1.1.2",
		"brightmax": 100,
		"brightmin": 10,
		"campere": 20,
		"category": "0505",
		"cmod": "rgbcw",
		"colormax": 100,
		"colormin": 10,
		"crc": 118,
		"cwmaxp": 100,
		"cwtype": 0,
		"defbright": 100,
		"defcolor": "c",
		"deftemp": 100,
		"dmod": 2,
		"iicb": 0,
		"iicc": 4,
		"iicg": 1,
		"iicr": 2,
		"iicscl": 8,
		"iicsda": 26,
		"iicw": 3,
		"module": "WB3L",
		"onoffmode": 0,
		"pmemory": 1,
		"rstbr": 50,
		"rstcor": "c",
		"rstnum": 3,
		"rsttemp": 100,
		"title20": 1,
		"wampere": 50,
		"wfcfg": "prod"
	}
