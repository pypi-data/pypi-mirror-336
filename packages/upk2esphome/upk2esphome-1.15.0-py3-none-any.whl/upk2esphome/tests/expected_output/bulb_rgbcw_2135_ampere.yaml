sm2135:
  clock_pin: P8
  data_pin: P26
  rgb_current: 20mA
  cw_current: 50mA

output:
  - platform: sm2135
    id: output_red
    channel: 2
  - platform: sm2135
    id: output_green
    channel: 1
  - platform: sm2135
    id: output_blue
    channel: 0
  - platform: sm2135
    id: output_cold
    channel: 4
  - platform: sm2135
    id: output_warm
    channel: 3

light:
  - platform: rgbww
    id: light_rgbww
    name: Light
    color_interlock: true
    cold_white_color_temperature: 6500 K
    warm_white_color_temperature: 2700 K
    red: output_red
    green: output_green
    blue: output_blue
    cold_white: output_cold
    warm_white: output_warm
